### ISSUE TITLE
generated base controller wrong namespace in findModel

### ISSUE THREAD

**Issue by daronjay** [cebe/yii2-openapi#147]
Hi, I have looked online and in the docs, cant see this issue. When I generate with ./yii gii/api --openApiPath=http://localhost:3000/documentation/json

My base classes all have a broken findModel method.  I am getting

`
    public function findModel($id)
    {
        **$model =  \::findOne($id);**
        if ($model !== null) {
            return $model;
        }
        throw new \yii\web\NotFoundHttpException("Object not found: $id");
    }`

where I would be expecting say  **\common\models\Company::findOne($id).**

I assume some config isn't set right, but I'm blowed if i can see how.

**Comment by daronjay** [comment]
So I have: 

```
generators =>   [
      'api' => [
        'class' => \cebe\yii2openapi\generator\ApiGenerator::class,
        'urlConfigFile' => '@api/config/url-rules.rest.php',
        'controllerNamespace' => 'api\controllers',
        'modelNamespace' => 'common\models',
        'fakerNamespace' => 'common\models\faker',
        'migrationPath' => '@common/migrations',
    ]
  ],

```

**Comment by SOHELAHMED7** [comment]
> We are running php 8.2.3, wondering if its some incompatibility?

Yes it is incompatible. But [the PR for PHP 8.2 compatibility is completed and tests are passing](https://github.com/SOHELAHMED7/yii2-openapi/pull/22) and new version of this lib is not released yet.

Steps to run [that version](https://github.com/SOHELAHMED7/yii2-openapi/pull/22) of this lib in your project using PHP 8.2.* can be found at https://github.com/SOHELAHMED7/yii2-openapi/blob/master/CONTRIBUTING.md#use-pr-of-your-own-fork-of-this-library-in-your-project-to-check-new-changes

Feel free to write us back with your experience and whether your issue is fixed.

**Comment by SOHELAHMED7** [comment]
This issue can be closed.
